android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -37,6 +38,9 @@ import kotlinx.coroutines.launch
 /** Helper class to bind to an interactor's lifecycle to translate it to a [Worker] lifecycle. */
 public object WorkerBinder {
 
+  internal val Worker.bindingCoroutineContext: CoroutineContext
```

Comment:
Can't it be `private` ? I don't see it being used in any other file

Reply:
Oh, guess it's used in the listener

Reply:
it can't, given the main bind method is defined outside WorkerBinder